src/module.c

```diff
@@ -857,7 +857,7 @@ void moduleCallCommandUnblockedHandler(client *c) {
     moduleCreateContext(&ctx, module, REDISMODULE_CTX_TEMP_CLIENT);
     selectDb(ctx.client, c->db->id);
 
-    CallReply *reply = moduleParseReply(c, &ctx);
+    CallReply *reply = moduleParseReply(c, NULL);
```

Comment:
so the effect here is that RM_CreateStringFromCallReply will create strings without auto-memory, and RM_FreeCallReply will not bother to try to free them?
anything else that uses this context?

Reply:
The effect is only if the user uses auto memory. And if he does then he set it on the unblock handler after the CallReply was already created so I do not think he should expect it to work on the provided CallReply. Regardless I think that the auto memory is a dangerous and flaky feature that should be deprecate ...